src/pages/project/project_wizard/CreateProject.tsx

```diff
@@ -1,26 +1,45 @@
-import { Box } from '@mui/material';
+import { useState } from 'react';
 import { useTranslation } from 'react-i18next';
+import { useSelector } from 'react-redux';
+import { useFetchCurrentInstitution } from '../../../api/hooks/useFetchCurrentInstitution';
 import { PageHeader } from '../../../components/PageHeader';
-import { StyledPageContent } from '../../../components/styled/Wrappers';
+import { StyledPageContent, WizardStartPageWrapper } from '../../../components/styled/Wrappers';
+import { RootState } from '../../../redux/store';
+import { emptyProject, SaveCristinProject } from '../../../types/project.types';
 import { EmptyProjectForm } from './EmptyProjectForm';
 import { NFRProject } from './NFRProject';
+import { ProjectForm } from './ProjectForm';
 
 const CreateProject = () => {
   const { t } = useTranslation();
+  const user = useSelector((store: RootState) => store.user);
+  const topOrgCristinId = user?.topOrgCristinId ?? '';
+  const currentInstitutionQuery = useFetchCurrentInstitution(topOrgCristinId);
+  const [newProject, setNewProject] = useState<SaveCristinProject>({ ...emptyProject });
+  const [showProjectForm, setShowProjectForm] = useState<boolean>(false);
```

Comment:
Etter å ha sett i child components her, ser det kanskje ut som at denne er overflødig, og i praksis bare skal speile verdien til `newProject` 🤔 
```suggestion
  const showProjectForm = !!newProject;
```

Reply:
Bra forslag! Jeg prøvde det og det funker ikke siden newProject aldri egentlig er tom (den starter med empty project), jeg kunne sjekket på tittel eller noe, men da begynner det å føles litt hacky. Dette er ganske clean, selv om det er litt overtydelig.